packages/ckeditor5-dev-ci/bin/notifycirclestatus.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+const allowedBranches = require( '../lib/data/allowedBranches.json' );
+const formatMessage = require( '../lib/formatMessage' );
+const printLog = require( '../lib/printLog' );
+const slackNotify = require( 'slack-notify' );
+
+const ALLOWED_EVENTS = [
+	'api',
+	'webhook'
+];
+
+// This script assumes that is being executed on Circle CI.
+// Step it is used on should have set value: `when: on_fail`, since it does not
+// check whether the build failed when determining whether the notification should be sent.
+// Described environment variables starting with "CKE5" should be added by the integrator.
+
+const {
+	/**
+	 * @enum {String} Token to a Github account with the scope: "repos". It is required for obtaining an author of
+	 * the commit if the build failed. The repository can be private and we can't use the public API.
+	 */
+	CKE5_GITHUB_TOKEN,
+
+	/**
+	 * @enum {String} Required.
+	 */
+	CKE5_CIRCLE_TOKEN,
+
+	/**
+	 * @enum {String} Required. Webhook URL of the Slack channel where the notification should be sent.
+	 */
+	CKE5_SLACK_WEBHOOK_URL,
+
+	/**
+	 * @enum {String} Required. Value of `pipeline.trigger_source`.
+	 * See: https://circleci.com/docs/variables/#pipeline-values.
+	 */
+	CKE5_TRIGGER_SOURCE,
+
+	/**
+	 * @enum {String} Optional. If both are defined, the script will use the URL as the commit URL.
+	 * Otherwise, URL will be constructed using current repository data.
+	 */
+	CKE5_TRIGGERING_REPOSITORY_SLUG,
+	CKE5_TRIGGERING_COMMIT_HASH,
```

Comment:
```suggestion
	TRIGGER_REPOSITORY_SLUG,
	TRIGGER_COMMIT_HASH,
```

I wonder if `TRIGGER_COMMIT` wouldn't be enough.

Reply:
Making the variable name more explicit does not hurt IMO, and it looks more in line with `REPOSITORY_SLUG` variable.